k8s/helm-charts/seldon-core-v2-crds/templates/seldon-v2-crds.yaml

```diff
@@ -370,8 +370,11 @@ spec:
                       type: string
                     type: array
                   joinType:
-                    description: One of inner (default), outer, or any (see above
-                      for details)
+                    default: inner
```

Comment:
descriptions are lost here compared to original, we should probably port them over in the source file (`operator/config/crd/bases/mlops.seldon.io_seldonruntimes.yaml`). I've done the same in #5291, but it's not merged in yet.